tests/openfl/utilities/test_change_tags.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (C) 2020-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+"""Data Splitters tests module."""
+import pytest
+
+from openfl.utilities import change_tags
+
+
+@pytest.mark.parametrize(
+    'tags,add_field,remove_field,expected_result', [
+        ('abc', None, 'abc', ()),
+        (('abc',), None, 'abc', ()),
+        (['abc'], None, 'abc', ()),
+        ('abc', 'def', None, ('abc', 'def')),
+        (['abc'], 'ghi', None, ('abc', 'ghi')),
+        (['abc', 'def'], 'ghi', None, ('abc', 'def', 'ghi')),
+        (['abc', 'def'], None, 'abc', ('def',)),
+        (['abc', 'def'], 'ghi', 'abc', ('def', 'ghi')),
+        (('abc', 'def'), 'ghi', 'abc', ('def', 'ghi')),
+        (('abc', 'def', 'def'), 'ghi', 'abc', ('def', 'ghi')),
+        (('abc', 'ghi', 'def'), 'ghi', 'abc', ('ghi', 'def')),
+    ])
+def test_equal(tags, add_field, remove_field, expected_result):
```

Comment:
Generally, test function should have name like `test_<tested_function_name>_<test_case>`.